### PR TITLE
chrony: update package description

### PR DESCRIPTION
**Pull request by regunakyle**
Maintainer: @mlichvar 
Compile tested: Not tested 
Run tested: Not tested 

Description:
Close this forum topic
[Suggestion: State the difference between chrony and chrony-nts in package description](https://forum.openwrt.org/t/suggestion-state-the-difference-between-chrony-and-chrony-nts-in-package-description/187392)

(First time doing pull request. Do I need to test this? I can spin up a VM if needed)

### CODE REVIEW

File: net/chrony/Makefile [hunk 1]
```diff
@@ -8,7 +8,7 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=chrony
-PKG_VERSION:=4.5
+PKG_VERSION:=4.6
```
Comment (efahl) [inline net/chrony/Makefile:11]: I think you've got the PKG_VERSION and PKG_RELEASE changes backwards, should be the same version, but the release gets bumped for non-functional changes.

Comment (regunakyle) [reply net/chrony/Makefile]: > I think you've got the PKG_VERSION and PKG_RELEASE changes backwards, should be the same version, but the release gets bumped for non-functional changes.
> 
> EDIT: oops, should also add that PKG_VERSION is dictated by upstream...

Thanks for the hint, fixed

**Comment by mlichvar** [comment]
Please squash the commits into one and add a Signed-off-by line. Other than that, it looks good to me. Thanks.

**Comment by regunakyle** [comment]
> Please squash the commits into one and add a Signed-off-by line. Other than that, it looks good to me. Thanks.

Done

**Pull request merged**